opal/mca/hwloc/hwloc191/Makefile.am

```diff
@@ -8,6 +8,13 @@
 # $HEADER$
 #
 
+# Due to what might be a bug in Automake, we need to remove stamp-h?
+# files manually.  See
+# http://debbugs.gnu.org/cgi/bugreport.cgi?bug=19418.
+DISTCLEANFILES = \
+    hwloc/include/hwloc/autogen/stamp-h? \
+    hwloc/include/private/autogen/stamp-h?
```

Comment:
OK, but I suspect it's against the intent of `DISTCLEANFILES`, since automake probably thinks these are all explicit files and you're really passing it a shell glob expression.  If the AM folks add some additional checking in the future there's a chance we could get burned.  Leave it as-is for now, but it would be more future-proof to delete them via `distclean-local`: http://www.gnu.org/software/automake/manual/automake.html#Clean